src/databricks/labs/blueprint/tui.py

```diff
@@ -143,3 +143,9 @@ def question(self, text: str, default: str | None = None, **_) -> str:
                 return default
             return answer
         raise ValueError(f"not mocked: {text}")
+
+    def extend(self, patterns_to_answers: dict[str, str]):
```

Comment:
can you change it to return a copy of `MockPrompts`, but with these additional patterns?